README.md

```diff
@@ -146,47 +163,65 @@ be given with `DSSC_IMAGE_NAME`.
-  on = "push"
-  resolves = "Scan with Deep Security Smart Check"
-}
-
-action "Build image" {
-  uses = "docker://docker:stable"
-  args = ["build", "-t", "registry.example.com/my-project/my-image", "."]
-}
-
-action "Docker Login" {
-  uses = "actions/docker/login@master"
-  env = {
-    DOCKER_REGISTRY_URL = "registry.example.com"
-  }
-  secrets = [
-    "DOCKER_USERNAME",
-    "DOCKER_PASSWORD"
-  ]
-}
-
-action "Push image" {
-  needs = ["Build image", "Docker Login"]
-  uses = "actions/docker/cli@master"
-  args = "push registry.example.com/my-project/my-image"
-}
-
-action "Scan with Deep Security Smart Check" {
-  needs = "Push image"
-  uses = "docker://deepsecurity/smartcheck-scan-action"
-  secrets = [
-    "DSSC_SMARTCHECK_HOST",
-    "DSSC_SMARTCHECK_USER",
-    "DSSC_SMARTCHECK_PASSWORD",
-    "DSSC_IMAGE_PULL_AUTH"
-  ]
-  args = ["--image-name registry.example.com/my-project/my-image"]
-}
+## Example Workflow Using Github Actions
+
+```yml
+name: Deep Security Smart Check
+
+on: 
+  push:
+    branches: 
+      - master
+      
+jobs:      
+    SmartCheck-Scan-Action:
+       runs-on: ubuntu-latest
+       steps:
+
+       # AWS Example:
+        - name: Deep Security Smart Check Scan ECR
+          uses: deepsecurity/Deep-Security-Smart-Check@version*
+          with:
+            DSSC_IMAGE_NAME: myECRrepo/myimage
+            DSSC_SMARTCHECK_HOST: ${{ secrets.DSSC_SMARTCHECK_HOST }}
+            DSSC_SMARTCHECK_USER: ${{ secrets.DSSC_SMARTCHECK_USER }}
+            DSSC_SMARTCHECK_PASSWORD: ${{ secrets.DSSC_SMARTCHECK_PASSWORD }}
+            DSSC_IMAGE_PULL_AUTH: ${{ secrets.DSSC_IMAGE_PULL_AUTH }}
+            DSSC_FINDINGS_THRESHOLD: '{"malware": 999, "vulnerabilities": { "defcon1": 999, "critical": 999, "high": 999 }, "contents": { "defcon1": 999, "critical": 999, "high": 999 }, "checklists": { "defcon1": 999, "critical": 999, "high": 999 }}'
+            DSSC_INSECURE_SKIP_TLS_VERIFY: true
+            DSSC_INSECURE_SKIP_REGISTRY_TLS_VERIFY: true
+
+        # Azure Example:
+        - name: Deep Security Smart Check Scan ACR
+          uses: deepsecurity/Deep-Security-Smart-Check@version*
+          with:
+            DSSC_IMAGE_NAME: myrepo.azurecr.io/myimage
+            DSSC_SMARTCHECK_HOST: ${{ secrets.DSSC_SMARTCHECK_HOST }}
+            DSSC_SMARTCHECK_USER: ${{ secrets.DSSC_SMARTCHECK_USER }}
+            DSSC_SMARTCHECK_PASSWORD: ${{ secrets.DSSC_SMARTCHECK_PASSWORD }}
+            DSSC_IMAGE_PULL_AUTH: '{"username": "${{ secrets.ACR_USER }}","password": "${{ secrets.ACR_PASSWORD }}"}'
+            DSSC_FINDINGS_THRESHOLD: '{"malware": 999, "vulnerabilities": { "defcon1": 999, "critical": 999, "high": 999 }, "contents": { "defcon1": 999, "critical": 999, "high": 999 }, "checklists": { "defcon1": 999, "critical": 999, "high": 999 }}'
+            DSSC_INSECURE_SKIP_TLS_VERIFY: true
+            DSSC_INSECURE_SKIP_REGISTRY_TLS_VERIFY: true
+```
+
+## Example Workflow Running a Docker Container
+
```

Comment:
This was a little confusing for me at first. It might be worth mentioning that the Github action merely wraps the docker container, and that the container can be used by itself like so.

Reply:
That's a good one, maybe at the start of the README we can add something like:

---

### This project was built by the [Deep Security Smart Check](trendmicro.com/smartcheck) team to help you to scan your containers in your CI/CD pipeline.

### You can use as a standalone Docker container published in the [Dockerhub](https://hub.docker.com/r/deepsecurity/smartcheck-scan-action) to scan your images. This tool is also used by the [Deep Security Smart Check plugin for Jenkins](https://plugins.jenkins.io/deepsecurity-smartcheck/) and the GitHub Action, that wraps the container published in Dockerhub.

---